### DISCUSSION TITLE
ServiceWorker registration failed

### DISCUSSION THREAD

**Discussion by samplics** [firefly-iii/firefly-iii#6914] [Firefly III]
My installation seems to work fine, haven't found anything that doesn't work, but there is always this concerning error in the console.

```The path of the provided scope ('/firefly-iii/public') is not under the max scope allowed ('/firefly-iii/public/'). Adjust the scope, move the Service Worker script, or use the Service-Worker-Allowed HTTP header to allow the scope.```

I am using cloudflare and have their origin server certificate installed on my server.
I have also done all of the .env configurations that are recommended.

**Comment by JC5** [comment]
Is Firefly III running under `/firefly-iii/public/` when you go to visit it?

**Reply by samplics** [reply]
Yes

**Reply by JC5** [reply]
Alright fix that first, that's pretty insecure.

**Reply by samplics** [reply]
I'm not too familiar with PHP and Apache2, should I just move the contents of the directory to the base folder? `/var/www/html`

**Reply by samplics** [reply]
Alright I figured it out, setting `DocumentRoot /var/www/html/firefly-iii/public` in my apache config did the trick!

**Reply by JC5** [reply]
Nice, that should do it! Keep in mind the service worker will be removed at some point anyway.
